src/components/TextInput/TextInputClearButton/index.tsx

```diff
@@ -40,4 +40,4 @@ function TextInputClearButton({onPressButton}: TextInputClearButtonProps) {
 
 TextInputClearButton.displayName = 'TextInputClearButton';
 
-export default forwardRef(TextInputClearButton);
```

Comment:
this is unrelated but a source of warning in the browser console. While working on updated React I wanted to get rid of as many console warnings as I can since it helps to find the actual React bugs

Reply:
Sounds good, but may I suggest creating a separate PR to remove this ref?